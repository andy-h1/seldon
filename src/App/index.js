import React from 'react';
import { PageDashboard } from '../PageDashboard';

const App = () => {
  return <PageDashboard />;
};

export default App;
