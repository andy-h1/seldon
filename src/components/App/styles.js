import { createGlobalStyle } from 'styled-components';
import { makeStyles } from '@material-ui/core';
import { colours } from '../../tokens';

export const GlobalStyle = createGlobalStyle`

body {
    background-color: ${colours.darkGrey};
    color: ${colours.white};
    margin: 0;
    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
    sans-serif;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
}

code {
  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
    monospace;
}
`;

export const useStyles = makeStyles(() => ({
  root: {
    flexGrow: 1,
  },
}));
