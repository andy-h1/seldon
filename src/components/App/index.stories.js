import React from 'react';
import App from '.';

export default {
  title: 'App',
  component: App,
};

export const Template = (args) => <App {...args} />;
