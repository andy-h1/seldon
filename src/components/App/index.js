import React from 'react';
import { Grid } from '@material-ui/core';
import * as S from './styles';
import { SidePanel } from '../SidePanel';
import { Header } from '../Header';

const App = () => {
  return (
    <>
      <S.GlobalStyle />
      <Grid container spacing={3}>
        <Grid item xs={3} direction="column" justify="center" alignItems="flex-start">
          <SidePanel />
        </Grid>
        <Grid item xs={9}>
          <Header />
        </Grid>
      </Grid>
    </>
  );
};

export default App;
