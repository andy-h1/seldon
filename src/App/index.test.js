import React from 'react';
import { render, waitFor } from '@testing-library/react';
import App from '.';

describe('App component', () => {
  it('should render', async () => {
    const { container } = render(<App />);
    await waitFor(() => expect(container.firstChild).toMatchSnapshot());
  });
});
