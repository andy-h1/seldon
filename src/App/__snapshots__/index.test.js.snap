// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render 1`] = `
<div>
  <h1>
    Dashboard
  </h1>
  <div>
    <svg />
  </div>
</div>
`;
